source/_integrations/fitbit.markdown

```diff
@@ -13,13 +13,43 @@ ha_integration_type: integration
 
 The Fitbit sensor allows you to expose data from [Fitbit](https://fitbit.com/) to Home Assistant.
 
+## Requirements before installation
+
 Prior to enabling the sensor, your Home Assistant instance must be enabled with SSL, using services such as [Home Assistant cloud](/cloud/) or [Duck DNS](/integrations/duckdns/) integrating Let's Encrypt. Fitbit will not allow insecure integrations.
 
+It is necessary to have configured the internal and external URLs. Settings-> System-> Network → (Home Assistant URL)
+For example something like this if you have configured Duck DNS changing to the corresponding values.
+- Internet: https://CustomName.duckdns.org:8123
+- Local Network https://192.168.x.xxx:8123
+
 <div class='note'>
 There is no need for Access from the Internet to Home Assistant for this Integration to work. Home Assistant can only be accessible from the internal Network or via VPN. But a Certificate is required in any case.
 </div>
 
-Your Fitbit account must also be registered as an Developer account at the [Fitbit Developer Portal](https://dev.fitbit.com), and have a verified email address. 
+## Create app on Fitbit
```

Comment:
This can be made afterwards, and part of the data needed is provided during the setup process after this. At this point and state the integration is in, it makes no sense to provide this as a prerequisite IMHO.